Core/EventHandlers.lua

```diff
@@ -1987,6 +1987,20 @@ function R:OnLootReady(event, ...)
 			end
 		end
 
+		--Handling for Winter Night Dreamsaber mount
+		if Rarity.isOpening and Rarity.lastNode == L["Dreamseed Cache"] then
+			Rarity:Debug("Detected Opening on " .. Rarity.lastNode .. " (method = SPECIAL)")
+			local v = self.db.profile.groups.mounts["Reins of the Winter Night Dreamsaber"]
+			if v and type(v) == "table" and v.enabled ~= false then
```

Comment:
You can (and probably should) use `addAttemptForItem` here, so that future refactoring will be easier.